node-graph/gcore/src/raster.rs

```diff
@@ -532,7 +532,7 @@ fn dimensions_node<_P>(input: ImageSlice<'input, _P>) -> (u32, u32) {
 #[cfg(feature = "alloc")]
 pub use image::{CollectNode, Image, ImageFrame, ImageRefNode, MapImageSliceNode};
 #[cfg(feature = "alloc")]
-mod image {
+pub(crate) mod image {
```

Comment:
We should probably move this to another file, but we can do that in another pr if you prefer